posts/2021-06-11-kubernetes-operator-with-java-and-quarkus.adoc

```diff
@@ -0,0 +1,712 @@
+= Kubernetes Operator with Java and Quarkus
+
+== The goal of this blog post
+A Kubernetes operator is an application that runs
+Even though Kubernetes provides extensive support for multiple container management operations, it doesn't cover all potential cases related to application management. Some applications need the manual intervention of a human operator. A Kubernetes operator is an application that runs in the cluster and manages resources.
+It allows transferring of operational knowledge of the human operator into software components. Operators can be used for multiple purposes such as restoring data, creating cloud resources, resilience testing, and many others.
+
+The purpose of this blog post is to demonstrate how to quickly write and deploy a simple Kubernetes operator using Java, and provide an example of tasks that can be performed using an operator.
+
+==== Why Java?
+
+* Being able to develop operators in Java can bring a lot of benefits for companies and individuals that are focused mainly on the Java stack.
+* Quarkus with the main focus on cloud-native, and microservice solutions looks like a good fit for building an operator.
```

Comment:
I'd re-word this sentence a bit. Since you are under the "Why Java" section. So this seems a bit out of place or not tied into the main point. Also, as someone not super familiar with operators, why is something with cloud-native and microservice solutions a good fit?

Reply:
I changed the heading to 
`==== The benefits of choosing Java and Quarkus`
to make the items tight to the main point.

Also, add some extra explanation regarding why cloud-native is a good fit.
`Technologies such as containers, microservices, immutable infrastructure, deployed via declarative code are common elements of cloud-native computing approach. It is a good fit for writing an operator which needs to be deployed in a container, and the state of resources managed by an operator is defined in a declarative way. Quarkus with the main focus on cloud-native, and microservice solutions looks like a good fit for building an operator.`